src/main/java/com/rtb/projectmanagementtool/task/TaskTreeData.java

```diff
@@ -0,0 +1,64 @@
+package com.rtb.projectmanagementtool.task;
+
+import java.util.ArrayList;
+
+/** Class containing task data. */
+public final class TaskTreeData {
+  /** Enum containing status options for a task. */
+  public enum Status {
+    COMPLETE,
+    INCOMPLETE
+  }
+
+  private TaskData task;
+  private ArrayList<TaskTreeData> subtasks;
+
+  public TaskTreeData(TaskData task, ArrayList<TaskTreeData> subtasks) {
+    this.task = task;
```

Comment:
In this case since you have both the parent and the child tasks in the same structure, it would be clearer if you could explicitly label the parentTask